apps/ui/src/views/Townhall/Topics.vue

```diff
@@ -91,10 +93,17 @@ watchEffect(() => setTitle(`Topics - ${props.space.name}`));
       <UiTooltip title="New topic">
         <UiButton
           :to="{
-            name: 'space-townhall-create',
-            params: { space: `${space.network}:${space.id}` },
-            query: {
-              category: categoryId
+            name: category
```

Comment:
Similar case that can be rolled into single ternary (as we already did it for one above).